cloud/getting-started/about-centreon-cloud.md

```diff
@@ -0,0 +1,40 @@
+---
+id: about-centreon-cloud
+title: About Centreon Cloud
+---
+
+## What is Centreon Cloud?
+
+Centreon Cloud is the Centreon monitoring platform installed in our Cloud infrastructure. For detailed product information, visit our [product description page](https://www.centreon.com/centreon-cloud/).
```

Comment:
Should rather be: 
Centreon Cloud is the Centreon monitoring platform delivered as a service and hosted and managed by Centreon from our Cloud infrastructure.